tools/filter_compounds/filter_compounds.py

```diff
@@ -0,0 +1,61 @@
+import re
+import argparse
+
+from openbabel import openbabel, pybel
+openbabel.obErrorLog.StopLogging()
```

Comment:
This seems suspicious, why do we need to stop logging here?

Reply:
I adopted this approach from the chemicaltoolbox tools (all that use OpenBabel contain this line), bacause otherwise Galaxy crashes with error due to logs produced by OpenBabel (that were not actually errors).

Reply:
I doubt Galaxy crashes because of this. Do you mean the job will result in error because openbabel writes non-errors to stderr?

Reply:
Yes, I meant the job. I tried to remove the lines in my testing galaxy environment and the job went through, which is strange because I remember quite clearly that before Christmas the job failed and adding the line suppressing logging helped (also, some warnings about compound structure that OpenBabel quite commonly shows is now part of the stderr output, but the job is nevertheless successful). I tested also the Remove counterions and fragments tool I modified before that is part of Chemicaltoolbox, removed the `.StolLogging()` and it works as well. So I am not really sure what happed here...